docs/self-managed/deployment/helm/configure/authentication-and-authorization/generic-oidc-provider.md

```diff
@@ -1,146 +1,784 @@
+:::
+
+## Configure Camunda components
+
+Configure Camunda components to use your OIDC provider through Helm values.
+
+### Global OIDC configuration
+
+Start with the global configuration that applies to all components:
+
+```yaml
+global:
+  security:
+    authentication:
+      method: oidc
+
+  identity:
+    auth:
+      enabled: true
+      type: "GENERIC"
+
+      # OIDC Provider Endpoints (from discovery document)
+      issuer: <issuer-url>
+      publicIssuerUrl: <issuer-url>
+      issuerBackendUrl: <issuer-url-or-internal-dns>
+      tokenUrl: <token-endpoint-url>
+      jwksUrl: <jwks-endpoint-url>
+```
+
+**Parameter descriptions:**
+
+| Parameter          | Description                                             | Example                                           |
+| ------------------ | ------------------------------------------------------- | ------------------------------------------------- |
+| `issuer`           | Canonical issuer URL (must match `iss` claim in tokens) | `https://login.example.com`                       |
+| `publicIssuerUrl`  | Issuer URL accessible from users' browsers              | `https://login.example.com`                       |
+| `issuerBackendUrl` | Issuer URL accessible from Kubernetes pods              | `http://oidc-internal.svc.cluster.local`          |
+| `tokenUrl`         | Token endpoint from discovery document                  | `https://login.example.com/oauth/token`           |
+| `jwksUrl`          | JWKS endpoint for token signature verification          | `https://login.example.com/.well-known/jwks.json` |
+
+:::tip When to use different issuer URLs
+For most deployments, all three issuer parameters have the same value (your OIDC provider's issuer URL).
+
+Use different values only when:
+
+- Your OIDC provider has separate internal and external endpoints
+- You're using Ingress with different internal/external DNS names
+- You want pods to use cluster-internal DNS for better performance
+
+**Example scenario:** External users access your provider at `https://login.example.com`, but Kubernetes pods can reach it faster via `http://oidc-provider.namespace.svc.cluster.local:8080`. In this case:
+
+- `issuer` and `publicIssuerUrl`: `https://login.example.com`
+- `issuerBackendUrl`: `http://oidc-provider.namespace.svc.cluster.local:8080`
+  :::
+
+### Configure Management Identity
+
+Add configuration for Management Identity:
+
+```yaml
+global:
+  identity:
+    auth:
+      identity:
+        clientId: <identity-client-id>
+        audience: <identity-audience>
+        redirectUrl: <identity-base-url>
+        secret:
+          existingSecret: oidc-credentials
+          existingSecretKey: identity-client-secret
+        initialClaimName: <user-claim-name>
+        initialClaimValue: <admin-user-claim-value>
+
+identity:
+  enabled: true
+  contextPath: "/identity" # Optional: adjust if using custom context path
```

Comment:
By setting the context path, the redirect URL for the localhost setup from the table above will not be correct, or?

Is there any particular reason that you are including it here? I wonder if we could remove it to keep the config minimal.